go.mod

```diff
@@ -50,6 +51,7 @@ require (
 	github.com/hashicorp/hc-install v0.8.0
 	github.com/hashicorp/terraform-config-inspect v0.0.0-20240607080351-271db412dbcb
 	github.com/hashicorp/terraform-exec v0.21.0
+	github.com/jackc/pgx/v5 v5.7.1
```

Comment:
This is what I added. From what I can see this is the best-maintained postgres driver for Go. The other changes to this file were done by `go mod tidy`.